extensions/amp-sidebar/0.1/test/test-amp-sidebar.js

```diff
@@ -68,6 +68,21 @@ describes.realWin('amp-sidebar 0.1 version', {
           return {iframe, ampSidebar};
         });
       });
+
+      it('should replace text to screen reader \
+      button in data-close-button-aria-label', () => {
+        return getAmpSidebar({'closeText':
+          'data-close-button-aria-label'}).then(obj => {
+            const sidebarElement = obj.ampSidebar;
+            const closeButton = sidebarElement.lastElementChild;
+            if (options.closeText) {
```

Comment:
@rhapsodyai the following lines:
```
            if (options.closeText) {
               obj.ampSidebar.setAttribute('data-close-button-aria-label',
                   options.closeText);
             };
```
should be moved to the `getAmpSidebar` function definition (line 44 of this file). you can put it right after the `        if (options.open) {
          ampSidebar.setAttribute('open', '');
        }` line.

A bit of background:  `getAmpSidebar` actually creates the sidebar component and since tests may want different configurations of the sidebar component, it already takes an `option` argument.